themes/default/content/blog/pulumi-release-notes-m59/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "July 28 releases: K8s GitOps, autonaming in Google Native"
+date: 2021-07-28T08:00:00-07:00
+draft: false
+meta_desc: "Kubernetes is faster, K8s operator supports GitOps, new resources and functionality for Azure and Google Cloud, and easier getting started in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+Another 3 weeks have passed, so it's time for another edition of the Pulumi release notes! This iteration we've been busy smashing bugs across our products (over 100!), but we've still got some great new features to share with you as well:
+
+- New and updated cloud providers
+  - [Kubernetes operator supports GitOps workflow]({{< relref "/blog/pulumi-release-notes-m59#kubernetes-operator-supports-gitops-workflow" >}})
+  - [Kubernetes improved performance]({{< relref "/blog/pulumi-release-notes-m59#kubernetes-improved-performance" >}})
+  - [12 new resources in the Azure Native provider]({{< relref "/blog/pulumi-release-notes-m59#12-new-resources-in-the-azure-native-provider" >}})
+  - [Google Native: autonaming; easier management of project and region settings]({{< relref "/blog/pulumi-release-notes-m59#google-native-autonaming-easier-management-of-project-and-region-settings" >}})
+- Pulumi CLI and core technologies
+  - [Automation API: Select your backend]({{< relref "/blog/pulumi-release-notes-m59#automation-api-select-your-backend" >}})
+- Pulumi Service and Pulumi.com
+  - [Get started with Pulumi, directly from the Pulumi Service]({{< relref "/blog/pulumi-release-notes-m59#get-started-with-pulumi-directly-from-the-pulumi-service" >}})
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Kubernetes operator supports GitOps workflow
+
+For some Kubernetes workflows, developers want to be able to store their configuration in a Git repository but ensure that their cluster automatically stays up-to-date with the latest configuration when it's checked in. With the latest update to the Pulumi Kubernetes Operator, you can set up this type of GitOps workflow. To get started, [specify the fully-qualified branch name](https://github.com/pulumi/pulumi-kubernetes-operator/blob/80398a85958215a7c2c87e9ce30c69998f6cdba9/pkg/apis/pulumi/v1alpha1/stack_types.go#L99-L101) that you want to track in the `Stack` CustomResource (e.g., `refs/heads/main` will track the `main` branch). The operator will track the specified branch and automatically deploy new commits that are pushed to the branch.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes-operator/issues/50)
```

Comment:
Perhaps add a code snippet of a Stack that uses this?

The Kubernetes YAML below will tell the Pulumi Kubernetes Operator to poll the `production` branch of the target GitHub repo, and deploy an update to the `acmecorp/s3-op-project/production` stack when there are changes.

```yaml
apiVersion: pulumi.com/v1alpha1
kind: Stack
metadata:
  name: s3-bucket-stack
spec:
  stack: acmecorp/s3-op-project/production
  projectRepo: https://github.com/acmecorp/test-s3-op-project
  branch: production 
  config:
    aws:region: us-east-2
```